utils/docker/Makefile

```
@@ -23,7 +23,7 @@ DOCKER_DEV_BUILD_OPTS ?=
 ifdef DOCKER_DEV_IMAGE_APT_ADD__PKGS
 	DOCKER_DEV_BUILD_OPTS += --build-arg APT_ADD_PKGS=$(DOCKER_DEV_IMAGE_APT_ADD__PKGS)
 endif
-
+DOCKER_DEV_BUILD_OPTS += --build-arg GIT_USER_NAME=$(shell git config user.name) --build-arg GIT_USER_EMAIL=$(shell git config user.email)
```

needed because we're doing the apply inside of docker, where git is unconfigured? 

kind of obvious, but could use a comment.

---

Needed but alas buggy: You have to quote these variables ....
```suggestion
DOCKER_DEV_BUILD_OPTS += --build-arg GIT_USER_NAME="$(shell git config user.name)" --build-arg GIT_USER_EMAIL="$(shell git config user.email)"
```